src/applications/vaos/components/NoValidVAFacilities.jsx

```diff
@@ -1,28 +1,88 @@
 import React from 'react';
-import { Link } from 'react-router';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LocationDirectionsLink from 'applications/facility-locator/components/search-results/LocationDirectionsLink';
+import { titleCase } from '../utils/appointment';
+
+export default function NoValidVAFacilities({
+  systemId,
+  typeOfCare,
+  systemDetails,
+}) {
+  const attributes = systemDetails?.attributes;
+  const address = attributes?.address?.physical;
+  const phone = attributes?.phone;
+  const hours = attributes?.hours;
 
-export default function NoValidVAFacilities({ systemId }) {
   return (
     <div aria-atomic="true" aria-live="assertive">
       <AlertBox
         status="warning"
-        headline="Sorry, we couldn't find any locations for you"
+        headline={`We’re sorry. This location doesn’t allow online scheduling for ${typeOfCare} appointments`}
       >
-        Some types of care are not available at certain VA locations and not all
-        VA locations have enabled online scheduling. You can choose a different
-        VA health system or{' '}
-        <a
-          href={`/find-locations/facility/vha_${systemId}`}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          contact the VA medical center you chose
-        </a>{' '}
-        to get help with the care you need.
         <p>
-          <Link to="/">Go back to VA Online Scheduling home.</Link>
+          You can’t schedule this type of appointment online. Please call the
+          medical center for more information.
         </p>
+        {systemDetails ? (
+          <div className="vads-u-padding-left--2 vads-u-border-left--4px vads-u-border-color--primary">
+            <span className="vads-u-font-weight--bold">{attributes?.name}</span>
+            <br />
+            <span>{address?.address1}</span>
+            <br />
+            {!!address?.address2 && (
+              <>
+                <span>{address?.address2}</span>
+                <br />
+              </>
+            )}
+            <span>
+              {address?.city}, {address?.state} {address?.zip}
+            </span>
+            <br />
+            <LocationDirectionsLink location={systemDetails} hideIcon />
+            <div className="vads-u-display--flex vads-u-margin-top--2">
+              <span className="vads-u-font-weight--bold">Hours:</span>
+              <ul className="usa-unstyled-list vads-u-margin-top--0">
+                {hours &&
+                  Object.keys(hours).map((day, index) => (
+                    <li key={`day-${index}`}>{titleCase(day)}</li>
+                  ))}
+              </ul>
+              <ul className="usa-unstyled-list vads-u-margin-top--0">
+                {hours &&
```

Comment:
The hours data we get from the facility locator is often bad or missing, should make sure we're handling those cases the same way as the FL app, if you haven't checked already. I believe there's still a Sentry error somewhere in the FL code that fires when the data is bad.